src/Routing/Router.php

```diff
@@ -746,7 +746,8 @@ public static function reverseToArray($params)
             $params['requested'],
             $params['return'],
             $params['_Token'],
-            $params['_matchedRoute']
+            $params['_matchedRoute'],
+            $params['_name']
```

Comment:
@markstory  I think output of this method should contain  `_name` like before

Reply:
Do you think it is useful for reverse routing?

Reply:
I never used this method , so I really don't know :)
I just say , this can be break BC, but I'm not sure
if you say :+1:  i say too :+1:

Reply:
I tested in local 
before and after these  changes `reverseToArray` returns same result